src/main/java/mate/academy/service/AuthenticationService.java

```diff
@@ -11,6 +14,20 @@ public class AuthenticationService {
      * Return false in any other cases.
      */
     public boolean login(String email, String password) {
-        return false;
+
+        User currentUser = UserService.findByEmail(email);
+        boolean isPasswordEqual = false;
+
+        for (User user : UserService.users) {
```

Comment:
you don't need to have access to `users` field in UserService

`findByEmail()` method already returns user by email, you only need to check if this user is not null and if password is the same as this user's password